diff --git a/node_modules/react-native-radio-btn/lib/radioButton.js b/node_modules/react-native-radio-btn/lib/radioButton.js
index 5d86953..d51537d 100644
--- a/node_modules/react-native-radio-btn/lib/radioButton.js
+++ b/node_modules/react-native-radio-btn/lib/radioButton.js
@@ -67,9 +67,9 @@ export default class RadioButton extends Component {
               this.props.style,
               this.props.isSelected
                 ? {
-                    backgroundColor: "#E08036",
+                    backgroundColor: "#006bcc",
                     borderWidth: 1,
-                    borderRadius: 5,
+                    borderRadius: 7,
                     borderColor: "#E08036",
                     margin: 4,
                     color: "#fff !important"
@@ -77,10 +77,19 @@ export default class RadioButton extends Component {
                 : {
                     backgroundColor: "#F3F3F3",
                     borderWidth: 1,
-                    borderRadius: 5,
+                    borderRadius: 7,
                     borderColor: "#E3E3E3",
                     margin: 4
-                }
+                },
+                this.props.isSelected?
+                ((this.props.displayText == "No")?{
+                    backgroundColor: "red",
+                    borderWidth: 1,
+                    borderRadius: 7,
+                    borderColor: "#E08036",
+                    margin: 4,
+                    color: "#fff !important"
+                }:{}):{},{alignItems:"center",justifyContent:"center"}
             ]}
           >
             <View
@@ -89,39 +98,9 @@ export default class RadioButton extends Component {
               <View
                 style={{
                   display: "flex",
-                  flexDirection: "row"
+                  flexDirection: "row",
                 }}
               >
-                {this.props.prefixWithAlphabet ? (
-                  <Text
-                    style={{
-                      color: `${
-                        this.props.isSelected
-                          ? this.props.prefixActiveColor
-                          : this.props.prefixColor
-                      }`,
-                      fontSize: 16,
-                      textTransform: "uppercase"
-                    }}
-                  >
-                    {alphabets[this.props.index]}.
-                  </Text>
-                ) : null}
-                {this.props.prefixWithNumbers ? (
-                  <Text
-                    style={{
-                      color: `${
-                        this.props.isSelected
-                          ? this.props.prefixActiveColor
-                          : this.props.prefixColor
-                      }`,
-                      fontSize: 16
-                    }}
-                  >
-                    {this.props.index + 1}.
-                  </Text>
-                ) : null}
-
                 <Text
                   style={{
                     color: `${
@@ -130,7 +109,7 @@ export default class RadioButton extends Component {
                         : this.props.displayTextColor
                     }`,
                     fontSize: 16,
-                    marginLeft: 8
+                    // backgroundColor:"pink"
                   }}
                 >
                   {this.props.displayText}
@@ -162,7 +141,8 @@ let styles = StyleSheet.create({
   },
   buttonText: {
     padding: 20,
-    color: "white"
+    color: "white",
+    textAlign:"center"
   },
   container: {
     flexGrow: 1,
@@ -174,14 +154,12 @@ let styles = StyleSheet.create({
     justifyContent: "center"
   },
   item: {
-    marginLeft: 10,
     alignItems: "center",
     justifyContent: "center",
     fontSize: 20,
     color: "#fff"
   },
   itemSelected: {
-    marginLeft: 10,
     alignItems: "center",
     justifyContent: "center",
     fontSize: 20,
